test/drive.js

```diff
@@ -128,6 +149,41 @@ test('Multiple Archives File', function (t) {
   })
 })
 
+test('Multiple Archive POST File', function (t) {
+  var onrequest = hyperdriveHttp(getArchive)
+  server.once('request', onrequest)
+  fs.createReadStream(path.join(__dirname, 'drive.js'))
+  .pipe(request.post('http://localhost:8000/', function (err, res, body) {
```

Comment:
Just want to make sure I understand this. The "default" archive is `archive3` so you are posting to root and it defaults to that. 

I wanted to supported a similar idea with archive "defaults" for dat-publish. But we should just make sure to document it and add a test for the GET w/ root default. I can do that later today if you want me to merge this so you can keep working on dat.haus.